sgpt/client.py

```diff
@@ -58,7 +59,15 @@ def _request(
             timeout=REQUEST_TIMEOUT,
             stream=stream,
         )
-        response.raise_for_status()
+        try:
+            response.raise_for_status()
+        except requests.exceptions.HTTPError as http_err:
+            # Check if the status code is 403 (Forbidden) or 401 (Unauthorized)
+            if http_err.response.status_code == 403 or http_err.response.status_code == 401:
+                typer.secho("Invalid OpenAI API key.", fg="red")
```

Comment:
We can add more info for user, something like `f"Invalid OpenAI API key, update your config file: {cfg.SHELL_GPT_CONFIG_PATH}"`.